[2024-07-22T02:33:13.223+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: example_dynamic_task_mapping.add_one scheduled__2022-07-22T00:00:00+00:00 map_index=0 [queued]>
[2024-07-22T02:33:13.255+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: example_dynamic_task_mapping.add_one scheduled__2022-07-22T00:00:00+00:00 map_index=0 [queued]>
[2024-07-22T02:33:13.263+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2024-07-22T02:33:13.264+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 1
[2024-07-22T02:33:13.265+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2024-07-22T02:33:13.302+0000] {taskinstance.py:1300} INFO - Executing <Mapped(_PythonDecoratedOperator): add_one> on 2022-07-22 00:00:00+00:00
[2024-07-22T02:33:13.318+0000] {standard_task_runner.py:55} INFO - Started process 1825 to run task
[2024-07-22T02:33:13.351+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'example_dynamic_task_mapping', 'add_one', 'scheduled__2022-07-22T00:00:00+00:00', '--job-id', '599', '--raw', '--subdir', '/home/***/.local/lib/python3.7/site-packages/***/example_dags/example_dynamic_task_mapping.py', '--cfg-path', '/tmp/tmpcx6xf3ux', '--map-index', '0']
[2024-07-22T02:33:13.388+0000] {standard_task_runner.py:83} INFO - Job 599: Subtask add_one
[2024-07-22T02:33:13.560+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 599 for task add_one ([Errno 2] No such file or directory: '/opt/airflow/logs/dag_id=example_dynamic_task_mapping/run_id=scheduled__2022-07-22T00:00:00+00:00/task_id=add_one/map_index=0/attempt=1.log'; 1825)
[2024-07-22T02:33:13.609+0000] {local_task_job.py:208} INFO - Task exited with return code 1
[2024-07-22T02:33:13.856+0000] {taskinstance.py:2578} INFO - 0 downstream tasks scheduled from follow-on schedule check
